totalRP3/Core/UITools.lua

```diff
@@ -438,7 +414,7 @@ local function setTooltipForFrame(Frame, GenFrame, GenFrameAnch, GenFrameX, GenF
 		if GenFrameAnch then
 			Frame.GenFrameAnch = "ANCHOR_"..GenFrameAnch;
 		else
-			Frame.GenFrameAnch = "ANCHOR_TOPRIGHT";
```

Comment:
To ward off the question - intentional. All call sites seem to be specifying this parameter, and most of them set it to either "TOP" or "RIGHT". I've noticed no breakages with this change at all.